src/import/load-costume.js

```diff
@@ -50,6 +51,7 @@ const loadCostumeFromAsset = function (costume, costumeAsset, runtime) {
         imageElement.src = costumeAsset.encodeDataURI();
     }).then(imageElement => {
         costume.skinId = runtime.renderer.createBitmapSkin(imageElement, costume.bitmapResolution, rotationCenter);
+        costume.size = runtime.renderer.getSkinSize(costume.skinId);
```

Comment:
@cwillisf I think that because the imagedata has already been loaded in the previous promise, and the `_getBitmapSize` function in the renderer just uses `image.width/image.height`, I think the size will always be available at this time. I tried with a few bitmap skins and it looks like it is working right.